cilium/cmd/policy_trace.go

```diff
@@ -170,33 +146,28 @@ func init() {
 	policyTraceCmd.Flags().StringVarP(&dstK8sPod, "dst-k8s-pod", "", "", "Destination k8s pod ([namespace:]podname)")
 }
 
-func appendEpLabelsToSlice(epID string, labelSlice []string) []string {
-	ep, err := client.EndpointGet(epID)
+func appendIdentityLabelsToSlice(secID string, labelSlice []string) []string {
```

Comment:
Since you've touch this, I suggest you change the argument's orders to be simular to `append(slice, elements...)`

Reply:
Done.